src/inputs/Table/Table.js

```diff
@@ -412,6 +503,41 @@ Table.propTypes = {
    * If not defined, this button will not be rendered.
    */
   onExport: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the add row button.
+   * If not defined, this button will not be rendered.
+   */
+  onAddRow: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the delete row button.
+   * If not defined, this button will not be rendered.
+   */
+  onDeleteRow: PropTypes.func,
+  /*
+   * Function to update the label of the delete confirmation dialog body
+   */
+  buildDialogBodyLabel: PropTypes.func.isRequired,
+};
+
+const DEFAULT_LABELS = {
+  clearErrors: 'Clear',
+  label: 'undefinedTable',
```

Comment:
Is this label `undefinedTable` on purpose?

Reply:
DONE